api/server.spec.js

```diff
@@ -0,0 +1,84 @@
+const server = require('./server');
+const request = require('supertest');
+const db = require('../data/dbConfig');
+
+const testGame = {
+  title: 'Pacman', // required
+  genre: 'Arcade', // required
+  releaseYear: 1980, // not required
+};
+
+describe('POST /games', () => {
+  afterEach(async () => {
+    await db('games').truncate();
+  });
+
+  it('Should return 201 on success', async () => {
+    const response = await request(server)
+      .post('/games')
+      .send(testGame);
+    expect(response.status).toEqual(201);
+  });
+  it('Should return 422 with incomplete info', async () => {
+    const response = await request(server)
+      .post('/games')
+      .send({ title: 'Halo: Combat Evolved', releaseYear: 2001 });
+    expect(response.status).toBe(422);
+  });
+  it('Should return 405 if duplicate game is inserted', async () => {
+    await request(server)
+      .post('/games')
+      .send(testGame);
+    const response = await request(server)
+      .post('/games')
+      .send(testGame);
```

Comment:
It feels like you should be able to do this with something like a for loop, but that's what I jump to when I see consecutively repeated code. Can you think of any good ways to compartmentalize this? specifically, 
```javascript
await request(server)
  .post('/games')
  .send(testGame);
```